host_source.go

```diff
@@ -56,6 +56,7 @@ const (
 
 type cassVersion struct {
 	Major, Minor, Patch int
+	AdditionalNotation  string
```

Comment:
In Cassandra (and many Java-based projects), the bit after the dash is often called a qualifier or sometimes a suffix. In semantic versioning, it might be part of a pre-release segment or a build meta after a plus sign. However, Cassandra uses variations like snapshot, rc, etc., which are typically called qualifiers.

So, I suggest changing the name of the AdditionalNotation field to the Qualifier.